lib/Activity.py

```diff
@@ -817,7 +828,38 @@ def watch_next_wf(self, sessionid):
             outfile = os.path.join(sessions_dir, f"{wfid}-{stage}.yaml")
             with open(outfile, "w", encoding='UTF-8') as fhandle:
                 yaml.dump(cfgmaps, fhandle)
+            files = [file for file in os.listdir(self.media_dir) if "tar" in file]
```

Comment:
we might want to tighten this up a bit more, looking for _specifically_ '.tar$' or '.tar.gz$'